docs/decisions/0014-ui-backend-api.md

```diff
@@ -0,0 +1,47 @@
+# s3gw-ui backend API
+
+## Using POST instead of GET for some operations
+
+While we should strive to ensure that we are compliant with the RESTful API
+paradigm, in some instances this is not viable.
+
+One such instance is when we have a lot of information that needs to be provided
+to the API during a GET call. In such a situation it would make sense to rely on
+the GET request's body to pass the additional context we need, but unfortunately
+browsers tend to drop the body for GET requests before sending the request to
+the server.
```

Comment:
Reference to the specs here:

- https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/send
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET
- https://xhr.spec.whatwg.org/#the-send()-method